content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -185,6 +185,50 @@ echo useSomeLib(new Helper('some text'))
 
 Global Variables however, are resolved at runtime.
 
+=== Loading libraries dynamically
+
+As of version 2.7 of the _Pipeline: Shared Groovy Libraries_ plugin,
+there is a new option for loading (non-implicit) libraries in a script:
+a `library` step that loads a library _dynamically_, at any time during the build.
+
+If you are only interested in using global variables/functions (from the `vars/` directory),
+the syntax is quite simple:
```

Comment:
Meaningful integration with Declarative is still TBD. Pending that, the intended uses are just those already laid out in this PR—dynamic version selection; and loading a lib without prior registration.